### DISCUSSION TITLE
Is there a way to add Generated Columns in MySQL using schema generator?

### DISCUSSION THREAD

**Discussion by keletsomolefe** [mikro-orm/mikro-orm#1678] [Ideas]

**Comment by B4nan** [comment]
To simple question, simple answer: nope. 

What is that good for? How would you envision the support? Are there other ORMs that do support it? How?

At least provide some examples if you want to propose a feature. 

In my eyes this is blocked by knex not supporting it (https://github.com/knex/knex/issues/3987).

You can use `@Formula()` in MikroORM for similar experience:

https://mikro-orm.io/docs/defining-entities/#formulas
